src/pages/AppDetail.svelte

```diff
@@ -7,36 +7,41 @@
   import NotFound from "../components/NotFound.svelte";
   import Pill from "../components/Pill.svelte";
   import { TabGroup, Tab, TabContent } from "../components/tabs";
+  import Title from "../components/Title.svelte";
 
   export let params;
 
   const appDataPromise = getAppData(params.app);
 </script>
 
 <style>
-  .table-header {
-    @apply table-auto;
-    @apply my-4;
+  a {
+    text-decoration: none;
   }
-  .table-header td {
-    @apply border;
-    @apply p-2;
+  .mzp-u-data-table {
+    margin-top: $spacing-md;
+    margin-bottom: $spacing-lg;
+    td {
+      border: 1px solid $color-light-gray-40;
+      padding: 0.5rem;
+    }
   }
 </style>
 
 {#await appDataPromise then app}
-  <h1>{params.app}</h1>
+  <Title text={params.app} />
+
   {#if app.deprecated}
     <Pill message="Deprecated" bgColor="#4a5568" />
   {/if}
-  <p class="mt-2">{app.description}</p>
-  <table class="table-header">
+  <p>{app.description}</p>
+  <table class="mzp-u-data-table">
     <tr>
-      <td>Source code Url</td>
+      <td>Source Code URL</td>
```

Comment:
I had been meaning to remove the URL part of this, since it's redundant. May as well change it here, if we're gonna change it.

```suggestion
      <td>Source Code</td>
```